bms/bms_app/static/bms_app/js/style.js

```diff
@@ -0,0 +1,57 @@
+
+function checkusername()
+{
+  var userbox = document.getElementById('username');
+  var usermessage = document.getElementById('usermsg');
+  re = /^\w+$/;
+  if(!re.test(userbox.value)) 
+  {
+    usermessage.style.color = "#ff6666" ;
+    usermessage.innerHTML ="Username must contain only letters, numbers and underscores!";
```

Comment:
Why inner HTML? Need alternate option for this. 

Also, no error messages should be in JS statically.